Backpack-SwiftUI/PageIndicator/Classes/BPKPageIndicator.swift

```diff
@@ -43,6 +43,7 @@ public struct BPKPageIndicator: UIViewRepresentable {
     
     public func updateUIView(_ uiView: UIPageControl, context: Context) {
         uiView.isUserInteractionEnabled = false
+        uiView.hidesForSinglePage = true
```

Comment:
The SwiftUI PageIndicator was missing this, where in the UIKit one we do have.
This makes the indicator disappear when there's only on item inside (intended behaviour!)